### ISSUE TITLE
Templates missing skip navigation link (accessibility)

### ISSUE THREAD

**Issue by lexoyo** [silexlabs/Silex#1664]
All of our templates lack a skip navigation link (lien d'evitement). Required by WCAG 2.1 AA and EU directive 2019/882

This is a work to do in a template, you'll need to [open a template (edit in silex button)](https://www.silex.me/templates/) then modify it and then click "Contribute changes" in the site settings:

<img width="1307" height="1117" alt="Silex site settings, with a Contribute changes link" src="https://github.com/user-attachments/assets/b66782ea-c1a9-4cb6-814c-5185ad1f4fa7" />

cc @ceubri

**Comment by ceubri** [comment]
yes i'm working on it
